mocha-client-tests/addon/manifest.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "Mocha tests",
+  "version": "1.0",
+  "manifest_version": 2,
+  "description": "Check ",
+  "icons": {
+    "16": "images/icon-16.png"
+  },
+  "short_name": "MochaTest",
+  "background": {
+    "scripts": [
+      "background.js"
+    ]
+  },
+  "permissions": [
+    "activeTab",
+    "tabs",
+    "http://*/*",
+    "https://*/*",
+    "bookmarks",
+    "notifications",
+    "storage",
+    "<all_urls>"
+  ],
+  "browser_action": {
+    "default_icon": {
+      "19": "images/icon-19.png"
+    },
+    "default_title": "Mocha Test",
+    "default_popup": "popup.html"
+  },
+  "applications": {
```

Comment:
You can remove this section entirely. It's not needed for this example. It could be harmful if someone copy/pasted it because it makes the extension only compatible with Firefox, not Chrome.